cmd/agent/daemon/app/app.go

```diff
@@ -245,10 +260,19 @@ func (a *App) Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	containersClient, err := containers.NewClient(log, cgroupClient, a.cfg.ContainerdSockPath)
+	procHandler := proc.New()
+	containersClient, err := containers.NewClient(log, cgroupClient, a.cfg.ContainerdSockPath, procHandler)
 	if err != nil {
 		return err
 	}
+	processTreeCollector, err := processtree.New(log, procHandler, containersClient)
```

Comment:
If there are no configured proc tree exporters we can skip process tree setup and init